Duplicate Beast IDs (Comma in name and Rate Variation)

Before you look at this in more detail, let me check whether this came about due to my configparser shenannigans – a very similar file did work a moment ago. But the fact that Beastling *can* output this, even though probably after being given a messy configuration object, is disturbing.

Beast complains about a `beastling`-generated XML:
```
IDs should be unique. Duplicate id 'featureClockRate:lexicon:flat' found

Error detected about here:
  <beast>
      <run id='mcmc' spec='MCMC'>
          <state id='state'>
              <parameter id='featureClockRate:lexicon:flat' name='stateNode'>
```

The ini file is

```
[admin]
basename = lessersunda_noproto_ratevariation_autocode

[languages]
exclusions = p-aust1307-abvd,p-aust1307-acd,p-cent2245-abvd,p-cent2245-acd,p-mala1545-abvd,p-mala1545-acd,hama1240,p-alor1249,p-timo1261,p-east2519

[model lexicon]
model = covarion
data = auto_data.tsv
file_format = cldf
rate_variation = True
```

There is currently a test for this, resulting in
```
AssertionError: Beast run on ('admin', 'mk', 'cldf_data_with_comma', 'rate_var') returned non-zero exit status 1
```


What is the clean test-driven development/nosetest/travis way to say “This test is a reminder of documented buggy behaviour, continue running it but don't consider its failure a deal-breaker for the software”?

@xrotwang – You know this stuff far better than I do, what do you suggest?

--- a/tests/beastrun_tests.py
+++ b/tests/beastrun_tests.py
@@ -60,6 +60,7 @@ def test_basic():
         ("admin", "mk", "random"),
         ("admin", "mk", "feature_with_comma"),
         ("admin", "mk", "cldf_data_with_comma"),
+        ("admin", "mk", "cldf_data_with_comma", "rate_var"),
         ("admin", "mk", "calibration"),
         ("admin", "mk", "calibration_by_iso"),
         ("admin", "mk", "calibration_nested"),

--- a/tests/data/cldf_with_comma.csv
+++ b/tests/data/cldf_with_comma.csv
@@ -1,5 +1,5 @@
 Language_ID,Feature_ID,Value
-aal,"f0, 1",1
+aal,"f1, 1",1
 aal,f1,1
 aal,f2,1
 aal,f3,1
@@ -9,7 +9,7 @@ aal,f6,?
 aal,f7,1
 aal,f8,?
 aal,f9,1
-aas,"f0, 1",2
+aas,"f1, 1",2
 aas,f1,2
 aas,f2,2
 aas,f3,1
@@ -19,7 +19,7 @@ aas,f6,?
 aas,f7,?
 aas,f8,1
 aas,f9,3
-kbt,"f0, 1",3
+kbt,"f1, 1",3
 kbt,f1,3
 kbt,f2,1
 kbt,f3,1
@@ -29,7 +29,7 @@ kbt,f6,?
 kbt,f7,2
 kbt,f8,?
 kbt,f9,5
-abg,"f0, 1",4
+abg,"f1, 1",4
 abg,f1,2
 abg,f2,2
 abg,f3,1
@@ -39,7 +39,7 @@ abg,f6,?
 abg,f7,?
 abg,f8,3
 abg,f9,4
-abf,"f0, 1",5
+abf,"f1, 1",5
 abf,f1,1
 abf,f2,1
 abf,f3,1

